Gateway/Request/AuthorizationRequest.php

```diff
@@ -47,6 +58,13 @@ public function build(array $buildSubject)
         $payment = $paymentDO->getPayment();
         $cc_exp_month = $payment->getAdditionalInformation("cc_exp_month");
         $cc_exp_year = $payment->getAdditionalInformation("cc_exp_year");
+        // phpcs:disable
```

Comment:
What's is this comment for? Is this needed?
Same for below.

Reply:
@mstein-ck ,

Yes, these comments are needed to disable warning messages about coding standards.
This comment is used to disable below warning message : 
**Line exceeds 120 characters; contains 173 characters**

**Please check this screenshot**: `https://tinyurl.com/2qq8jazb`